.circleci/config.yml

```diff
@@ -256,6 +256,38 @@ jobs:
               create $CIRCLE_TAG --name $CIRCLE_TAG \
               --publish ./dist/*
 
+  pynwb-dev-python37:
+    docker:
+      - image: circleci/python:3.7.0-stretch
+    steps:
+      - checkout
+      - run: git submodule sync
+      - run: git submodule update --init
+      - run:
+          <<: *initialize-venv
+      - run:
+          name: Install dev branch of PyNWB and re-install hdmf
+          command: |
+            . ../venv/bin/activate
+            git clone https://github.com/NeurodataWithoutBorders/pynwb.git
+            cd pynwb
+            pip install -U -rrequirements-dev.txt -rrequirements.txt
+            pip check
+            python setup.py develop
+            cd ..
+            pip uninstall hdmf==1.2.0
+            python setup.py develop
```

Comment:
``pip uninstall hdmf==1.2.0`` uninstalling the pinned version is dangerous in that it means you will need to update the test any time PyNWB changes the version of HDMF it is using

Reply:
Good point.